operator/internal/manifests/gateway_tenants.go

```diff
@@ -56,22 +56,29 @@ func ApplyGatewayDefaultOptions(opts *Options) error {
 	return nil
 }
 
-func configureDeploymentForMode(d *appsv1.Deployment, mode lokiv1beta1.ModeType, flags FeatureFlags) error {
+func configureDeploymentForMode(d *appsv1.Deployment, mode lokiv1beta1.ModeType, flags FeatureFlags, name, ns string) error {
```

Comment:
nit. The code base refers to the stack identity fields using variable/argument names as `stackName` and `stackNs`.